_episodes/045-sean-arthur.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Hyper with Sean McArthur"
+date: 2021-11-05T02:11:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e044-sean-mcarthur.mp3
+duration: "1:05:18"
+length: "63525368"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Sean McArthur](https://twitter.com/seanmonstar), the creator of [Hyper](https://github.com/hyperium/hyper), an HTTP library for Rust.
+
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps 
+- [@01:37] - The history of Hyper
+- [@07:41] - Is Hyper a client and server side component?
+- [@11:09] - [Async/Await](https://rust-lang.github.io/async-book/03_async_await/01_chapter.html)
+- [@13:24] - Any benefit using Async over Bloc?
```

Comment:
Should this say "over blocking"?
Also, async (and await) shouldn't be capitalized here or elsewhere.